### ISSUE TITLE
The bold text inside a walkthrough description is not being read out or shown in the accessible view exactly as is

### ISSUE THREAD

**Issue by Achyuth-ibm-z** [microsoft/vscode#247130]
The bold text inside a VSCode walkthrough description is not being read out or shown in the accessible view exactly as is. In order to make a text or a statement or a sentence bold inside a vscode walkthrough description the developer has to prefix it and also append it with ** . Now this ** is not visible to a user who is using the walkthrough and instead views them in bold style. But the NVDA screen reader isn't either reading it or displaying it as bold but instead reading and displaying the ** as is.

Attached the screenshot of how NVDA displays the ** as is in the Accessible View (the text in blue color starting from **watsonx Code).

- VS Code Version: 1.99.3
- OS Version:  Microsoft Windows 11 Enterprise 10.0.26100 N/A Build 26100

How to replicate:
1. Open VS Code and turn on screen reader accessibility mode
2. Open any walkthrough step that contains bold text inside it's description
3. Open the accessible view with "option + f2" on mac or "alt + f2" on Windows
4. Notice how the NVDA screen reader isn't either reading it or displaying it as bold but instead reading and displaying the ** as is.

<img width="692" alt="Image" src="https://github.com/user-attachments/assets/c003c452-18c7-4377-a728-d9b9d1f89402" />
